.github/workflows/ci-workflow.yml

```diff
@@ -6,6 +6,13 @@ jobs:
   run_tests:
     runs-on: ubuntu-latest
     steps:
+      - name: Update Shellcheck
+        run: |
+          # Update shellcheck to the latest by downloading the binary (0.7.0 instead of default 0.4)
```

Comment:
let's have a script to do this. It's cleaner to run a script here than a block of code.

Reply:
Sure, I'll make a script for this